src/Tokenizer/Analyzer/ArgumentsAnalyzer.php

```diff
@@ -87,6 +87,16 @@ public function getArguments(Tokens $tokens, int $openParenthesis, int $closePar
 
     public function getArgumentInfo(Tokens $tokens, int $argumentStart, int $argumentEnd): ArgumentAnalysis
     {
+        static $skipTypes = null;
+
+        if (null === $skipTypes) {
+            $skipTypes = [T_ELLIPSIS, CT::T_CONSTRUCTOR_PROPERTY_PROMOTION_PUBLIC, CT::T_CONSTRUCTOR_PROPERTY_PROMOTION_PROTECTED, CT::T_CONSTRUCTOR_PROPERTY_PROMOTION_PRIVATE];
+
+            if (\defined('T_READONLY')) { // @TODO: drop condition when PHP 8.1+ is required
```

Comment:
In `src/Fixer/FunctionNotation/NullableTypeDeclarationForDefaultNullValueFixer.php` you're checking `if (\PHP_VERSION_ID >= 70400)` and here there's only a check for defined keyword. I'm just curious if you'd have any standard for situations like this?

Maybe the one below would be easier to find in the future with required PHP 8.1?
```suggestion
            if (\PHP_VERSION_ID >= 80100) {
```

Reply:
I think the `defined` is maybe more correct here, as that is what we care about at this point in the code, however since PHP never adds new tokens in patches doing a version check would be effectively the same. 
The version check is a bit cheaper IIRC so that is probably why it was used in the other part of the fixer.

Reply:
Thanks for this explanation. Have a nice day!

Reply:
thanks for review :)  and have a great day as well!